roofit/histfactory/inc/RooStats/HistFactory/MakeModelAndMeasurementsFast.h

```diff
@@ -18,11 +19,12 @@
 namespace RooStats{
   namespace HistFactory{
 
-    RooWorkspace* MakeModelAndMeasurementFast( RooStats::HistFactory::Measurement& measurement );
-    //RooWorkspace* MakeModelFast( RooStats::HistFactory::Measurement& measurement );
+    RooWorkspace* MakeModelAndMeasurementFast(
```

Comment:
See? This name also didn't age well. 🙂

Reply:
Haha that's the worst name possible, and everyone uses it!